### DISCUSSION TITLE
How to create a hyperlink on a specific word in a cell and not on the whole cell using PhpSpreadsheet?

### DISCUSSION THREAD

**Discussion by modestguy** [PHPOffice/PhpSpreadsheet#3675] [Q&A] [answered]
For example: I have a cell with content: "text text2". And i want to make word "text2" refer to specific url (make hyperlink on word "text2"). How to do it using PhpSpreadsheet?

The documentation gives an example of a reference to the whole cell:

```
$spreadsheet->getActiveSheet()->setCellValue('E26', 'www.phpexcel.net');
$spreadsheet->getActiveSheet()->getCell('E26')->getHyperlink()->setUrl('https://www.example.com');
```

Thank you very much!

**Comment by MarkBaker** [comment, marked as answer]
Does MS Excel allow that?
If not, then the answer is that you can't.

I know that it can be done in Libre Office, but that feature isn't supported

**Reply by modestguy** [reply]
Oh sorry. I have no way to test in MS Excel, I tried in Libre. Then the question is closed, thanks for the help.
